fairlearn/metrics/_metric_frame.py

```diff
@@ -259,10 +259,10 @@ def __init__(
 
         # Add sensitive and conditional features to all_data
         for sf in sf_list:
-            all_data[sf.name_] = list(sf.raw_feature_)
+            all_data[sf.name_] = sf.raw_feature_
         if cf_list is not None:
             for cf in cf_list:
-                all_data[cf.name_] = list(cf.raw_feature_)
+                all_data[cf.name_] = cf.raw_feature_
```

Comment:
Related to [previous comment](https://github.com/fairlearn/fairlearn/pull/1546/files#r2049609262)